### ISSUE TITLE
Compare improvements forecasing model to baselevel

### ISSUE THREAD

**Issue by FrankKr** [alliander-opensource/AIFES#14]
Try different hyperparameters or loss functions or model types or weather data or ...
and see how the accuracy of the forecast changes.
